data_upload/extract.py

```diff
@@ -0,0 +1,103 @@
+"""Script to extract earthquake data from RDS and upload to S3 bucket as CSV"""
+import os
+import csv
+from datetime import datetime
+from dotenv import load_dotenv
+import boto3
+import psycopg2
+import psycopg2.extras
+import pandas as pd
+from psycopg2.extensions import connection, cursor
+
+QUERY = """ 
+SELECT *
+FROM earthquakes AS e
+JOIN alerts AS a ON e.alert_id = a.alert_id
+JOIN networks AS n ON e.network_id = n.network_id
+JOIN magnitude AS m ON e.magnitude_id = m.magnitude_id
+JOIN type AS t ON e.type_id = t.type_id;
+"""
+
+COLUMNS = [
+    'earthquake_id', 'time', 'magnitude', 'tsunami', 'felt_report_count', 'cdi',
+    'latitude', 'longitude', 'depth', 'detail_url', 'alert_type', 'magnitude_type',
+    'network_name', 'type_name']
+
+
+CSV_FILE = f"/tmp/{datetime.today().strftime('%Y-%m-%d')}_data.csv"
+
+
+def get_connection() -> connection:
+    """Gets connection to database"""
+    return psycopg2.connect(
+        user=os.getenv("DB_USER"),
+        password=os.getenv("DB_PASSWORD"),
+        host=os.getenv("DB_HOST"),
+        port=os.getenv("DB_PORT"),
+        database=os.getenv("DB_NAME")
+    )
+
+
+def extract_data() -> None:
+    """Extracts data from database and writes to a CSV file"""
+    conn = None
+    try:
+        conn = get_connection()
+        print("Executing query...")
```

Comment:
Maybe use logging rather than print?
```suggestion
        logging.info("Executing query...")
```